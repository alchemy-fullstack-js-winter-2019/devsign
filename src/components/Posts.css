.Posts {
  margin: auto;
 li {
    color: green;
    font-size: 1.25em;
    list-style-type: none;
    font-weight: bold;
  }
}