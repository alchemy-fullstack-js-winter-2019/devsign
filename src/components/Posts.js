import React from 'react';
import Header from './Header';
import Footer from './Footer';

export default function Posts() {
  return (
    <>
     <Header/>
    <section>
      <h1>List of Posts</h1>
      <ul>
        <li>Post#1 Title</li>
        <li>Post#2 Title</li>
        <li>Post#2 Title</li>
      </ul>
    </section>
    <Footer/>
    </>
  );
}
