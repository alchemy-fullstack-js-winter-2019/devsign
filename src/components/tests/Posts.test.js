import React from 'react';
import renderer from 'react-test-renderer';
import Posts from '../Posts';
import { MemoryRouter } from 'react-router';

describe('Posts snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Posts />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
