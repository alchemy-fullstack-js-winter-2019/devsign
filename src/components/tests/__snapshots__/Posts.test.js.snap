// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts snapshot can match a snapshot 1`] = `
Array [
  <p>
    My Blog
  </p>,
  <nav
    className="Header"
  >
    <span>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </span>
    <span>
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </span>
    <span>
      <a
        href="/posts"
        onClick={[Function]}
      >
        Posts
      </a>
    </span>
    <span>
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </span>
  </nav>,
  <section>
    <h1>
      List of Posts
    </h1>
    <ul
      className="Posts"
    >
      <li>
        Post#1 Title
      </li>
      <li>
        Post#2 Title
      </li>
      <li>
        Post#2 Title
      </li>
    </ul>
  </section>,
  <p
    className="Footer"
  >
    Carmen Health Blog Copyright 2019
  </p>,
]
`;
